README.rst

```diff
@@ -62,8 +62,7 @@ You need to have precommit setup to use this hook.
    ::
 
         pip install pre-commit
-        pre-commit install
-        pre-commit install --hook-type commit-msg
+        pre-commit install --install-hooks
```

Comment:
This won't work. Need to explicitly run `pre-commit install --hook-type commit-msg` otherwise giticket wont work. needs to be reverted

Reply:
Looks like you're right. I expected that --install-hooks was minimally sufficient, but it doesn't actually install any hooks, just sets up the envs for the existing hooks.